src/ifcfg/parser.py

```diff
@@ -349,6 +349,8 @@ def default_interface(self):
         """
         return self._default_interface()
 
+    def alter(self, interfaces):
+        return interfaces
```

Comment:
Is this new method intentional?

Reply:
It is. It is there to maintain the consistency from ifconfig. Without it all of the flags would be lower case as opposed to the LinuxParser which returns them all in the original upper case.

Reply:
Understood, thanks for elaborating :+1: